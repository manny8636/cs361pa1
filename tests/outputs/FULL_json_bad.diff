--- outputs/FULL_json_bad.txt	2022-02-15 14:30:55.063071389 -0500
+++ expected/FULL_json_bad.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,3 @@
+SYNTAX ERROR: '}' is an invalid token
+SYNTAX ERROR: Could not process text beginning at ' }'
+Parsing ../data/json_bad.txt failed
