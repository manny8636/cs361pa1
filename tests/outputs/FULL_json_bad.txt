Parsing ../data/json_bad.txt failed
