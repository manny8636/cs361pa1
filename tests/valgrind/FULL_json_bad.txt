==372158== Memcheck, a memory error detector
==372158== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372158== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372158== Command: ../json -o ../data/json_bad.txt
==372158== 
==372158== 
==372158== HEAP SUMMARY:
==372158==     in use at exit: 2,064 bytes in 1 blocks
==372158==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372158== 
==372158== LEAK SUMMARY:
==372158==    definitely lost: 2,064 bytes in 1 blocks
==372158==    indirectly lost: 0 bytes in 0 blocks
==372158==      possibly lost: 0 bytes in 0 blocks
==372158==    still reachable: 0 bytes in 0 blocks
==372158==         suppressed: 0 bytes in 0 blocks
==372158== Rerun with --leak-check=full to see details of leaked memory
==372158== 
==372158== For lists of detected and suppressed errors, rerun with: -s
==372158== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
