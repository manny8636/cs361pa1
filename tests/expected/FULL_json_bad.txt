SYNTAX ERROR: '}' is an invalid token
SYNTAX ERROR: Could not process text beginning at ' }'
Parsing ../data/json_bad.txt failed
